relate/bin/relate.py

```diff
@@ -91,8 +91,31 @@ def expand_yaml(yml_file, repo_root):
 # }}}
 
 
-# {{{ code test
+# {{{ lint YAML
+def lint_yaml(args):
+    import os
+
+    from yamllint import linter
+    from yamllint.cli import show_problems
+    from yamllint.config import YamlLintConfig
+
+    conf = YamlLintConfig(file=args.config_file)
+
+    for root, _, filenames in os.walk(args.DIRECTORY):
+        for f in filenames:
+            filepath = os.path.join(root, f)
+            if not conf.is_file_ignored(f) and conf.is_yaml_file(f):
+                # expanded yaml is missing a newline at the end of the file
+                # which causes the linter to complain, so we add a newline :)
```

Comment:
Is that newline missing for all Jinja output?

Reply:
The newline is missing for any files that are processed by `expand_yaml`. The linter will complain that the yaml file is missing a newline at the end of the file.